src/site/content/en/fast/preload-critical-assets/index.md

```diff
@@ -137,6 +138,36 @@ import(_/* webpackPreload: true */_ "CriticalChunk")
 
 If you are using an older version of webpack, use a third-party plugin such as [preload-webpack-plugin](https://github.com/GoogleChromeLabs/preload-webpack-plugin).
 
+## Effects of preloading on Core Web Vitals
+
+Preloading is a powerful performance optimization that has an effect on loading speed. Such optimizations can lead to changes in your site's [Core Web Vitals](/vitals/), and it's important to be aware them.
+
+### Largest Contentful Paint (LCP)
+
+Preloading has a powerful effect on [Largest Contentful Paint (LCP)](/lcp/) when it comes to fonts and images, as both images and text nodes can be [LCP candidates](/lcp/#what-elements-are-considered). Hero images and large runs of text that are rendered using web fonts can benefit significantly from a well-placed preload hint, and should be used when there are opportunities to deliver these important bits of content to the user faster.
+
+However, you want to be careful when it comes to preloading&mdash;and other optimizations! In particular, avoid preloading too many resources. When a resource is preloaded, its [resource priority](/prioritize-resources/) is effectively raised from whatever its initial priority would have been to "High" or "Highest". If too many resources are prioritized, effectively none of them are. The effects of excessive preload hints will be especially detrimental to those on slower networks where bandwidth contention will be more evident.
+
+Instead, focus on a few high-value resources that you know will benefit from a well-placed preload. When preloading fonts, ensure that you're serving fonts in WOFF 2.0 format to reduce resource load time as much as possible. Since WOFF 2.0 has [excellent browser support](https://caniuse.com/woff2), using older formats such as WOFF 1.0 or TrueType (TTF) will delay your LCP if the LCP candidate is a text node.
+
+When it comes to LCP and JavaScript, you'll want to ensure that you're sending complete markup from the server in order for the [browser's preload scanner](/preload-scanner/) to work properly. If you're serving up an experience that relies entirely on JavaScript to render markup and can't send server-rendered HTML, it may be advantageous to preload JavaScript critical for rendering in order to keep your LCP as low as possible.
```

Comment:
> it may be advantageous to preload JavaScript critical for rendering

I think we want to say that it may be advantageous to preload resources that are only discoverable once the JavaScript finishes loading.